cmd/crictl/sandbox.go

```diff
@@ -192,6 +186,14 @@ var listPodSandboxCommand = cli.Command{
 			Name:  "output, o",
 			Usage: "Output format, One of: json|yaml|table",
 		},
+		cli.BoolFlag{
+			Name:  "latest, l",
+			Usage: "Show recently created sandboxes",
+		},
+		cli.IntFlag{
+			Name:  "last, n",
+			Usage: "Show last n recently created sandboxes",
+		},
```

Comment:
Seems two options are same. Merge last to latest and defaulting latest to one?

Reply:
Initially just want to keep same UX with docker, but of course we could make change.

Reply:
@feiskyer I tried to merge them into one flag. But I find that it's hard to set default for `-l`, because once you define it as an int flag, you have to specify a value when using it.

Given that I think it's fine to keep 2 flags as what docker does.

Reply:
OK, didn't notice this problem. let's keep both